### ISSUE TITLE
Path been 'doubled' 

### ISSUE THREAD

**Issue by kadsyy** [croogo/croogo#803]
Hi, i have successfully install croogo in my system just that some links are not correctly pointed. For example ;
![image](https://cloud.githubusercontent.com/assets/4100091/23940974/ec625ba8-09a1-11e7-9249-369e4a22af52.png)

at the breadcrumb the 'Themes' path are pointed to -
 `http://localhost/mysite/mysite/admin/extensions/extensions-themes`

same as the theme screenshot - 
`http://localhost/mysite/mysite/croogo/core/img/screenshot.png`

should be -
`http://localhost/mysite/croogo/core/img/screenshot.png`

-----
croogo3
php 5.6

**Comment by liviascapin** [comment]
I think maybe all the breadcrumbs as set in https://github.com/croogo/croogo/commit/324a5d03174c94e45759bd5011159fd14defdce7 need to be changed from
```php
$this->request->here();
´´´
to
```php
$this->request->here(false);
´´´
This will trim the base path because the base will already be set by Router::url?

**Comment by rchavik** [comment]
Fixed by #821
